test/cpp/end2end/client_callback_end2end_test.cc

```diff
@@ -350,13 +334,11 @@ class ClientCallbackEnd2endTest
 };
 
 TEST_P(ClientCallbackEnd2endTest, SimpleRpc) {
-  MAYBE_SKIP_TEST;
```

Comment:
Not really a review question, more for my sake - how to do you specify grpc_iomg_run_in_background to be false in order to test these tests with callback alternative? (might be nice to include in a comment if there's a specific way)

Reply:
Ultimately, it is resolved in the table associated with the specific event engine: https://github.com/grpc/grpc/blob/master/src/core/lib/iomgr/ev_posix.h#L49

There is no OSS event engine that runs in the background at the current time, which is why we didn't previously support callback API in OSS. This PR allows callback API to be used without a background iomgr.